pom.xml

```
@@ -8,23 +8,22 @@
     <relativePath />
   </parent>
   <artifactId>gitlab-plugin</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>1.7.1</version>
   <name>GitLab Plugin</name>
   <url>https://github.com/jenkinsci/${project.artifactId}</url>
   <packaging>hpi</packaging>
 
   <properties>
-    <revision>1.7.2</revision>
+    <revision>1.7.1</revision>
```

Don't want to go backwards in time with version numbers
```suggestion
    <revision>1.7.2</revision>
```

---

Sorry, there was a bug on GitHub before when I was trying to cut a new release, so unintentionally messed things up a bit.